examples/wagmi-v2/test/e2e.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { BrowserContext, expect, test as baseTest } from "@playwright/test";
+import dappwright, { Dappwright, MetaMaskWallet } from "@tenkeylabs/dappwright";
+
+export const test = baseTest.extend<{
+  context: BrowserContext;
+  wallet: Dappwright;
+}>({
+  context: async ({}, use) => {
+    // Launch context with extension
+    const [wallet, _, context] = await dappwright.bootstrap("", {
+      wallet: "metamask",
```

Comment:
No [Brave](https://github.com/TenKeyLabs/dappwright/blob/d7af3fa0e6aadf28928e9a8b25475cbc3c76cec6/src/wallets/wallets.ts#L2-L3) yet. We'd have to write that a little by hand.

Reply:
That's ok :D

For now this is awesome!